src/Target/DrushTarget.php

```diff
@@ -100,7 +100,7 @@ public function setUri(string $uri)
      */
     public function getAvailableTargets():array
     {
-      $aliases = $this['service.local']->run('drush site-alias --format=json', function ($output) {
+      $aliases = $this['service.local']->run('drush site:alias --format=json', function ($output) {
```

Comment:
Ideally we'd allow this to be backwards compatible with drush8, and perhaps check the version of drush running before changing the command.

It looks like drutiny does some version check here https://github.com/drutiny/drutiny/blob/3d91515650abba658f7d50bdcc163b6f49c52838/src/Driver/DrushRouter.php#L42 but maybe this needs to be abstracted so it can be used globally?